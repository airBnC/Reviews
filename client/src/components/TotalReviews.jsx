import React from 'react';

import SearchReviews from './SearchReviews.jsx';

const TotalReviews = props => (
  <div id="review-overall-average">
    <h6> 315 Reviews </h6>
    <h6> ***** </h6>
  </div>
);

export default TotalReviews;