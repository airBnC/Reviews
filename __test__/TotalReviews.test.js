import React from 'react';
import { configure, shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import TotalReviews from '../client/src/components/TotalReviews';

configure({ adapter: new Adapter() });

describe('TotalReviews', () => {
  it('should render correctly', () => {
    const output = shallow(<TotalReviews />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});

