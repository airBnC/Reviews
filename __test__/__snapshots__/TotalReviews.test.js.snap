// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TotalReviews should render correctly 1`] = `
<styled.span>
  <styled.span>
     
    1
     Reviews 
  </styled.span>
  <styled.span>
    <StarsOverall
      overallAvg={1}
    />
  </styled.span>
</styled.span>
`;
